docs/ref/models/database-functions.txt

```diff
@@ -117,6 +117,24 @@ Usage example::
     >>> print(author.name_lower)
     margaret smith
 
+Now
+---
+
+.. class:: Now()
+
+.. versionadded:: 1.9
+
+Returns the current date and time. Since this uses your database server's
```

Comment:
Returns the database server's... (and chop second sentence which is a bit confusing about what "synchronization purposes" menas IMO?)